Simplified/NYPLBookRegistryRecord.m

```diff
@@ -141,7 +141,7 @@ - (NSDictionary *)dictionaryRepresentation
   
   return @{BookKey: [self.book dictionaryRepresentation],
            LocationKey: NYPLNullFromNil([self.location dictionaryRepresentation]),
-           StateKey: NYPLBookStateToString(self.state),
+           StateKey: @"NYPLBookstatetodo",
```

Comment:
This doesn't look like one of the enum's defined keys. What does this do?

Reply:
Was trying to get the code compile, replaced with the method call